docs/consume-packages/install-use-packages-nuget-cli.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Install and manage NuGet packages using the nuget.exe CLI
+description: Instructions for using the nuget.exe CLI to work with NuGet packages.
+author: mikejo5000
+ms.author: mikejo
+ms.date: 06/03/2019
+ms.topic: conceptual
+---
+
+# Install and manage packages using the nuget.exe CLI
+
+The CLI tool allows you to easily install, uninstall, and update NuGet packages in projects and solutions. This tool provides all NuGet capabilities on Windows, and also provides most features on Mac and Linux when running under Mono.
+
+The nuget.exe CLI is for your .NET Framework project and non-SDK-style projects (for example, one that targets .NET Standard libraries). If you are using a non-SDK-style project that has been migrated to `PackageReference`, use the dotnet CLI instead. The NuGet CLI requires a [packages.config](../reference/packages-config.md) file for packages references.
+
+> [!NOTE]
+> In most scenarios, we recommend [migrating non-SDK-style projects](../reference/migrate-packages-config-to-package-reference.md) that use `packages.config` to PackageReference, and then you can use the dotnet CLI instead of the `nuget.exe` CLI. Migration is not currently available for C++ and ASP.NET projects.
+
+This article shows you basic usage for a few of the most common nuget.exe CLI commands. For most of these commands, the CLI tool looks for a project file in the current directory, unless a project file is specified in the command. For a complete list of commands and the arguments you may use, see the [nuget.exe CLI reference](../tools/nuget-exe-cli-reference.md).
+
+## Prerequisites
+
+- Install the `nuget.exe` CLI by downloading it from [nuget.org](https://dist.nuget.org/win-x86-commandline/latest/nuget.exe), saving that `.exe` file to a suitable folder, and adding that folder to your PATH environment variable.
+
+## Install a package
+
+The [install](../tools/cli-ref-install.md) command downloads and installs a package into a project, defaulting to the current folder, using specified package sources. Install new packages into the *packages* folder in your project root directory.
+
+The `install` command doesn't modify a project file or the `packages.config` unless you specify this in the `install` command using the `-OutputDirectory` option or `packages.config` as the value for the `-ConfigFile` option. (Alternatively, to add the package to your project dependencies, modify `packages.config`, then use either `install` or `restore`.)
```

Comment:
`nuget install` does not install a package into a project, unless I'm doing something wrong with my testing. It "installs" (unzips) into a directory, but does not modify the .NET project to use it.

Additionally the `-ConfigFile` option is used to specify a `nuget.config` file, not `packages.config`, so that different package sources can be used, for example. Trying to use `-ConfigFile packages.config` results in the error "NuGet.Config does not contain the expected root element: 'configuration'. Path: 'C:\git\test\simplePCProject\ConsoleApp1\packages.config'."

Reply:
On the second point, yes, good catch. It is supposed to be the configFilePath argument, not -ConfigFile.
https://review.docs.microsoft.com/en-us/nuget/tools/cli-ref-install?branch=pr-en-us-1467#usage

@nkolev92 Nikolche tells me you can specify the csproj so that the project gets updated correctly. Looking at the docs, looks like that is using -OutputDirectory. I haven't tested that command yet though.